src/components/UI/Button/Button.js

```diff
@@ -12,6 +12,7 @@ export const Button = ({
   height,
   icon,
   iconAlign = 'left',
+  buttonClass,
```

Comment:
If you added an option to send class prop, why `buttonClass` as single and not just `classes` as array?

Reply:
currently, that's the only class a button gets in the whole project.
also, there's an extended usage of a util function that gets an array of values and returns a string.